examples/lighting-app/mbed/CMakeLists.txt

```diff
@@ -14,7 +14,8 @@ configure_file(
     @ONLY
 )
 
-set(MBED_PATH $ENV{MBED_OS_PATH} CACHE INTERNAL "")
+#set(MBED_PATH $ENV{MBED_OS_PATH} CACHE INTERNAL "")
```

Comment:
Can we remove this line if it's commented out?

Reply:
ok done